tests/testthat/error/test-subscript-loc.txt

```diff
@@ -35,6 +35,20 @@ Error: Must subset elements with a valid subscript vector.
 x The subscript can't contain negative locations.
 
 
+num_as_location() optionally forbids zero indices
+=================================================
+
+> num_as_location(0L, 1L, zero = "error")
+Error: Must subset elements with a valid subscript vector.
+x The subscript can't contain `0` values.
+i It had a zero value at location 1.
```

Comment:
In the `x` bullet zero is numeric, in the `i` bullet it is textual. Use numeric everywhere?